lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -168,6 +177,18 @@ private void buildCompositionLayer() {
         this, Layer.Factory.newInstance(composition), composition.getLayers(), composition);
   }
 
+  private void applyColorFilters() {
+    if (compositionLayer == null) {
+      return;
+    }
+
+    ColorFilterData[] data = (ColorFilterData[]) colorFilterData.toArray();
```

Comment:
@seanfchan Why do you need to do toArray vs just iterating instead of
`for (int i = 0; i < colorFilterData.size(); i++)`?
toArray actually allocates a new array (https://android.googlesource.com/platform/libcore/+/refs/heads/master/ojluni/src/main/java/java/util/Arrays.java#3138)